src/components/typography/text/README.md

```diff
@@ -45,7 +45,7 @@ Wraps the given text in the given HTML header `size`.
 | Props         | Type             | Required | Values         | Default |
 | ------------- | ---------------- | :------: | -------------- | ------- |
 | `elementType` | `String`         |    ✅    | `['h4', 'h5']` |         |
-| `isBold`      | `Boolean`        |    ✅    |                | `false` |
+| `isBold`      | `Boolean`        |          |                | `false` |
```

Comment:
GitHub is getting ridiculously good at diffs. This prop was actually not required to my understanding.

Reply:
Sure thing. Done.

Reply:
Thanks! I created #27 to keep track of it.